src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -64,7 +64,10 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NetCoreAppNativeLibrary Include="@(NetCoreAppNativeLibrary->'%(Identity)')">
         <EscapedPath>$(IlcFrameworkNativePath)lib%(Identity).a</EscapedPath>
       </NetCoreAppNativeLibrary>
+      <NativeLibrary Include="$(NugetPackageRoot)/microsoft.dotnet.ilcompiler/7.0.0-dev/shim_source_code/libs/System.Globalization.Native/build/libSystem.Globalization.Native-Static.a" Condition="'$(LocalSystemGlobalizationNative)' != ''"/>
```

Comment:
@jkotas  I need your help to replace the hardcoded "7.0.0-dev"

Reply:
NugetPackageRoot is nuget cache. The nuget tool is the only one allowed to write into nuget cache.

This should get compiled to `IntermediateOutputPath` instead. It will also fix the hardcoded `7.0.0-dev`.

Reply:
Addressed in next revision using IntermediateOutputPath